config

```diff
@@ -898,14 +898,12 @@ fi
 
 OUT="$OUT"
 
-$PERL $THERE/Configure LIST | grep "$OUT" > /dev/null
-
 if [ "$OUT" = "darwin64-x86_64-cc" ]; then
     echo "WARNING! If you wish to build 32-bit libraries, then you have to"
     echo "         invoke 'KERNEL_BITS=32 $THERE/config $options'."
 fi
 
-if [ $? = "0" ]; then
+if $PERL $THERE/Configure LIST | grep "$OUT" > /dev/null; then
```

Comment:
You reversed the order of the Configure command and the warning, now the warning comes at the beginning instead of the end. This will make it more likely to be overlooked IMHO. How about just saving the error instead?

```bash
$PERL $THERE/Configure LIST | grep "$OUT" > /dev/null
result=$?

if [ "$OUT" = "darwin64-x86_64-cc" ]; then
    echo "WARNING! If you wish to build 32-bit libraries, then you have to"
    echo "         invoke 'KERNEL_BITS=32 $THERE/config $options'."
fi

if [ $result = "0" ]; then
...
fi

Reply:
Oh, forget my last comment, I overlooked the `> /dev/null`.